src/components/GenerateKeys/ConfirmGenerate.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Modal from '../Modal';
+import Button from '@/atoms/Button';
+import Warning from '@/molecules/Warning';
+
+interface Props {
+  handleContinue: () => void;
+  handleClose: () => void;
+}
+const list: string[] = [
```

Comment:
Can we give a more descriptive name for this variable? We already know that this is a list because it's an Array.